Content.Server/Salvage/SalvageSystem.Runner.cs

```diff
@@ -164,7 +166,16 @@ private void UpdateRunner()
             else if (comp.Stage < ExpeditionStage.MusicCountdown && remaining < TimeSpan.FromMinutes(2))
             {
                 // TODO: Some way to play audio attached to a map for players.
-                comp.Stream = _audio.PlayGlobal(comp.Sound, Filter.BroadcastMap(Comp<MapComponent>(uid).MapId), true);
+                // Parkstation-ExpeditionMusic Start
+                // comp.Stream = _audio.PlayGlobal(comp.Sound, Filter.BroadcastMap(Comp<MapComponent>(uid).MapId), true);
+                var sound = comp.Sound;
+                if (sound == null && _prototypeManager.TryIndex<SoundCollectionPrototype>("ExpeditionCountdownDefault", out var collection))
```

Comment:
Falling back to a hardcoded prototype ID in code isn't great, there should just be a normal default.